### PR TITLE
Move to flexbox columns, update conditionals logic, add shortcut icons

### PR DESCRIPTION
**Pull request by darkwing**
-  I saw that there were properties of 'hasActive' and 'hasInactive', so I didn't need to check run length for conditions.  

-  Chrome was showing the Google favicon for some reason, so I added Mozilla.org favicons

-  Floats and widths and margins are awful.  I'm switching to flexbox, which is a bit more code, but it's the future™

### CODE REVIEW

**Comment by pdehaan** [comment]
Hhmm, I'm only seeing a 1-column layout (FF Beta/36 and Chrome Release):
runs_ _loads_app

File: client/static/assets/stylus/app.styl [hunk 1]
```diff
@@ -51,23 +51,33 @@ h2 .label {
...
-  width: -webkit-calc(50% - 20px);
-  float: left;
-  margin-right: 20px;
+.two-column-row {
+  display: flex;
+  width: 100%;
   
-  &.clear {
-    clear: left;
+  @media (max-width: 768px) {
```
Comment (pdehaan) [inline client/static/assets/stylus/app.styl]: `768px` is used at least 3 times in this file. Not sure if it is worth moving into some variable. Probably not.

File: client/static/assets/views/_partials/runs-active-table.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
     <tr runs-head></tr>
   </thead>
   <tbody>
-    <tr runs-body ng-repeat="run in runs.active" id="run-{{ run.runId }}" class="{{ run.cssClass }}" ng-if="runs.active.length > 0"></tr>
-    <tr ng-if="runs.active.length == 0"><td>There are no active runs.</td></tr>
+    <tr runs-body ng-repeat="run in runs.active" id="run-{{ run.runId }}" class="{{ run.cssClass }}" ng-if="runs.hasActive"></tr>
+    <tr ng-if="!runs.hasActive"><td>There are no active runs.</td></tr>
```
Comment (pdehaan) [inline client/static/assets/views/_partials/runs-active-table.html]: Nice!

File: client/static/index.html [hunk 3]
```diff
@@ -13,6 +13,10 @@
   <link href="assets/css/app.css" media="all" rel="stylesheet" type="text/css" />
   <!-- endbuild -->
 
+  <link rel="shortcut icon" href="//mozorg.cdn.mozilla.net/media/img/favicon.ico">
+  <link rel="apple-touch-icon" type="image/png" sizes="180x180" href="//mozorg.cdn.mozilla.net/media/img/favicon/apple-touch-icon-180x180.png">
+  <link rel="icon" type="image/png" sizes="196x196" href="//mozorg.cdn.mozilla.net/media/img/favicon/favicon-196x196.png">
+
```
Comment (pdehaan) [inline client/static/index.html]: Looks good. not sure if you want to add to the client/static/404.html file as well.

**Comment by darkwing** [comment]
Updates made.  I was seeing everything correctly in Chrome release, and Firefox Dev Edition, but I saw one column in Safari so I fixed that with a `-webkit-` prefix.  I'm also seeing two columns in release Firefox too :/

**Comment by pdehaan** [comment]
:+1:
works for me. I was possibly being stupid and not regenerating the stylus->css before opening my big, stupid face.

**Pull request merged**